data/core.yaml

```diff
@@ -801,6 +801,11 @@ en:
     field_source: add source
     field_source_label: Source for {field_name}
     field_source_placeholder: URL, newspaper article, book...
+    source:
+      main_input: General source
+      name: Name of the source
```

Comment:
Since the subfields are already part of the Sources field, we don’t need to repeat “source” here. I don’t think users will confuse this with a field for the name of the feature.

On that note, we should rename the Sources field back to just “Source”, in anticipation of a different solution for tagging multiple sources. Otherwise, the user might think each line is for a different source. The placeholders would guard against that, but a placeholder doesn’t appear when the subfield has some text in it.

Reply:
I agree that the field should be named "Source" now and I've been trying to find where the field name is defined, but I wasn't able to.
I found the [preset definition in id-tagging-schema](https://github.com/OpenHistoricalMap/id-tagging-schema/blob/b947fd62cc17535b5b89b3f31cc92ac962ea90e4/data/fields/source.json#L6), but I believe this refers to the Sources field available after creating a changeset on OSM. It is possible that this preset is also used for the OHM Sources field because it is linked to the `source` key. Does it make sense to change it?

Reply:
The same preset is used in both situations. We have a general mechanism for providing (or overriding) strings used for individual fields or presets:

https://github.com/OpenHistoricalMap/iD/blob/5cfff589c5b6c2bbeca6bc0f2073149d99d7db6c/data/core.yaml#L2528-L2541

If you search for `custom_presets` in the codebase, you can see where we prioritize these strings over the ones that come from id-tagging-schema.